radiography/src/main/java/radiography/SkipIdsViewFilter.kt

```diff
@@ -0,0 +1,15 @@
+package radiography
+
+import android.view.View
+import java.util.Arrays
+
+/**
+ * Filters out views with ids matching [skippedIds] from the output of [Radiography.scan].
+ */
+class SkipIdsViewFilter(private vararg val skippedIds: Int) : ViewFilter {
+
+  override fun matches(view: View): Boolean {
+    val viewId = view.id
+    return (viewId == View.NO_ID || Arrays.binarySearch(skippedIds, viewId) < 0)
```

Comment:
FYI this copies the bug in #14 – not in scope for this change, we can fix after this lands.

Reply:
added `|| skippedIds.isEmpty()`

Reply:
I would think the `skippedIds.binarySearch` extension would handle this case.

Reply:
```
public fun IntArray.binarySearch(element: Int, fromIndex: Int = 0, toIndex: Int = size): Int {
    return java.util.Arrays.binarySearch(this, fromIndex, toIndex, element)
}
```

Reply:
unless the crash was an NPE on the array, in which case that can't happen anymore?